integration-cli/docker_cli_run_unix_test.go

```diff
@@ -370,3 +370,38 @@ func (s *DockerSuite) TestRunSwapLessThanMemoryLimit(c *check.C) {
 		c.Fatalf("Expected output to contain %q, not %q", out, expected)
 	}
 }
+
+// "1" should be printed
+func (s *DockerSuite) TestRunSysctls(c *check.C) {
+
+	out, _, err := dockerCmdWithError("run", "--sysctl", "net.ipv4/ip_forward=1", "--name", "test", "busybox", "cat", "/proc/sys/net/ipv4/ip_forward")
```

Comment:
Two things. This test doesn't compile, and seems it should be `net.ipv4.ip_forward=1`